docs/user-guide/pods/init-container.md

```diff
@@ -0,0 +1,168 @@
+---
+assignees:
+- erictune
+
+---
+
+* TOC
+{:toc}
+
+In addition to having one or more main containers (or **app containers**), a
+pod can also have one or more **init containers** which run before the app
+containers.   Init containers allow you to reduce and reorganize setup scripts
+and "glue code".
+
+## Summary
+
+An init container is exactly like a regular container, except that it always
+runs to completion and each init container must complete successfully before
+the next one is started. If the init container fails (exits with a non-zero
+exit code) on a `RestartNever` pod the pod will fail - otherwise it will be
```

Comment:
Change the order on the last two parts of the sentence.

"If the init container fails, Kubernetes will restart the pod until the init container succeeds. If a pod is marked as `RestartNever`, the pod will fail if the init container fails."